src/test/java/com/lob/client/test/AddressTest.java

```diff
@@ -27,6 +27,9 @@
     @Test
     public void testListAddresses() throws Exception {
         final AddressResponseList addresses = client.getAddresses().get();
+        assertEquals(200, addresses.getStatusCode());
```

Comment:
nit: for all of these, can you do `assertEquals(addresses.getStatusCode(), 200);` to avoid [yoda conditions](https://en.wikipedia.org/wiki/Yoda_conditions)?